server/src/main/java/org/elasticsearch/rest/action/cat/RestRepositoriesAction.java

```diff
@@ -79,7 +79,7 @@ protected Table getTableWithHeader(RestRequest request) {
 
     private Table buildTable(RestRequest req, GetRepositoriesResponse getRepositoriesResponse) {
         Table table = getTableWithHeader(req);
-        for (RepositoryMetaData repositoryMetaData : getRepositoriesResponse.repositories()) {
+        for (RepositoryMetaData repositoryMetaData : getRepositoriesResponse.repositories().repositories()) {
```

Comment:
`repositories().repositories()` looks very confusing. Maybe the GetRepositoriesResponse.repositories() should continue to return the List, internally calling repositories.repositories()?